src/plugins/telemetryTable/collections/TableRowCollection.js

```diff
@@ -124,10 +129,22 @@ export default class TableRowCollection extends EventEmitter {
     return foundIndex;
   }
 
-  updateRowInPlace(row, index) {
-    const foundRow = this.rows[index];
-    foundRow.updateWithDatum(row.datum);
-    this.rows[index] = foundRow;
```

Comment:
🤦 _this_ is [why reactivity was broken](https://vuejs.org/guide/essentials/list.html#mutation-methods). Good catch!